src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -12,16 +12,16 @@ Capturing transactions requires that you first <PlatformLink to="/performance/">
 
 ## UIViewController Instrumentation
 
-This feature is available for iOS, tvOS, and Mac Catalyst, and only works for UIViewControllers.
+This feature is available for iOS, tvOS, and Mac Catalyst, is enabled by default, and works for `UIViewControllers`.
 
-The UIViewController Instrumentation, once enabled, captures transactions when your app loads an in-app [UIViewController][UIViewController]. However, the SDK doesn't capture transactions for UIViewControllers of third-party libraries or SwiftUI.
+The `UIViewController` Instrumentation captures transactions when your app loads an in-app [`UIViewController`][UIViewController]. However, the SDK doesn't capture transactions for `UIViewControllers` of third-party libraries or SwiftUI.
```

Comment:
It's weird to pluralize `UIViewControllers`, because that's not what you would see/put in your code, so I made everything singular to get around that.

```suggestion
The `UIViewController` Instrumentation captures transactions when your app loads an in-app [`UIViewController`][UIViewController]. However, the SDK doesn't capture transactions for the `UIViewController` of a third-party library or SwiftUI.
```